akka-remote/src/main/java/akka/remote/artery/compress/CountMinSketch.java

```diff
@@ -143,34 +69,22 @@ public long addObjectAndEstimateCount(Object item, long count) {
       // parameter to constructor.
       throw new IllegalArgumentException("Negative increments not implemented");
     }
-    MurmurHash.hashBuckets(item, recyclableCMSHashBuckets, width);
+    Murmur3.hashBuckets(item, recyclableCMSHashBuckets, width);
     for (int i = 0; i < depth; ++i) {
       table[i][recyclableCMSHashBuckets[i]] += count;
     }
     size += count;
     return estimateCount(recyclableCMSHashBuckets);
   }
-  
+
   public long size() {
     return size;
   }
-  
-  /**
-   * The estimate is correct within {@code 'epsilon' * (total item count)},
-   * with probability {@code confidence}.
-   */
-  public long estimateCount(long item) {
-    long res = Long.MAX_VALUE;
-    for (int i = 0; i < depth; ++i) {
-      res = Math.min(res, table[i][hash(item, i)]);
-    }
-    return res;
-  }
```

Comment:
Ok. I will implement it in correct way, this one based on different `hash` method.